lib/desktop/init.dart

```diff
@@ -298,10 +305,9 @@ void _initLogging(List<String> argv) {
     }
   });
 
-  final logLevelIndex = argv.indexOf('--log-level');
-  if (logLevelIndex != -1) {
+  if (args['log-level'] != null) {
     try {
-      final levelName = argv[logLevelIndex + 1];
+      final levelName = args['log-level'];
```

Comment:
Move declaration of `levelName` to the start, and do the null check on that, instead of repeated access of args['log-level']:

```
final levelName = args['log-level'];
if (levelName != null) {
  ...
}
```
Same for args['log-file'] above.